metricbeat/helper/interfacer.go

```diff
@@ -32,6 +33,7 @@ type ModuleConfig struct {
 	MetricSets []string `config:"metricsets"`
 	Enabled    bool     `config:"enabled"`
 	Selectors  []string `config:"selectors"`
+	Filter     []filter.FilterConfig
```

Comment:
You don't have to add Filter under Metricbeat, it's part of libbeat: https://github.com/elastic/beats/blob/master/libbeat/beat/beat.go#L104